tileview/src/main/java/com/qozix/tileview/markers/MarkerLayout.java

```diff
@@ -16,6 +16,10 @@
 
   private MarkerTapListener mMarkerTapListener;
 
+  public static final int POS_RELATIVE = 0;               // Position is relative
+  public static final int POS_ABSOLUTE_LEFT_TOP = 1;      // Position is absolute calculated from left or top of marker
+  public static final int POS_ABSOLUTE_RIGHT_BOTTOM = 2;  // Position is absolute calculated from right or bottom of marker
```

Comment:
have you found it's necessary to have both the top/left and bottom/right flags?  it seems a little cleaner to _just_ have relative vs absolute...

Reply:
The reason I had this was that if you have for example a 9-patch and you want the center to be 50 pixels in from the right then there is no way to do this counting from the left as you don't know the width of the marker until it's actually displayed. Hence you need one flag from each direction.

However one could also choose to have a negative offset meaning it's calculated from the left/top and a positive offset meaning it's calculated from the right/bottom. A method like that however would prevent you from setting an absolute position outside the marker as an anchor point.

The current relative offsets are counted from the top left I think and they are always negative if you want the anchor to end up inside the bounds of the marker (ie. shifting the marker up or to the left in relation to the anchor point).

Which method would you prefer?

Reply:
it's late and i may not be thinking this through appropriately, but what if the relative and absolute values were combined?  e.g.,

1.0 relative and 10px offset = 10px from bottom right
0.0 relative and 10px offset = 10px from top left
-1.0 relative and 10px offset = (width + 10px from left, height + 10px from top)
-0.5 x relative and 0px x offset, -1.0 y relative and -20px y offset = would be centered horizontally, and 20px above the point...

etc...

thoughts?

Reply:
That... is a very good idea. Simply get rid of the flags and have both a relative position and an absolute position for the anchor point. I'll fix that and submit again.